### DISCUSSION TITLE
Conditions in a field that is in a group

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#16158] [Archive]
Am I missing something? Or is it a bug? Pay attention to the video - I have 3 conditions and one of them does not work when the field is inside a group and it concerns the starting value. All I have to do is move the field out of the group and all the conditions work.
I have the latest version of directus, but that also applies to previous versions.

https://user-images.githubusercontent.com/90528895/197756209-9cb228ec-28fe-4031-9913-456e7450d5f4.mp4

**Comment by koksikus** [comment]
One more thing, look at the first box field on the right under the group which starts with the word "Czynnosc_" the three dots are at the end, not right after the text.

**Reply by connorwinston** [reply]
@koksikus I am not able to reproduce this, can you try the following for me.

1. Update to the latest version and try again.
2. Under the Conditions > Interface Options...clear the value so it uses the field defaults. This would help me narrow down what is causing this.
<img width="712" alt="image" src="https://user-images.githubusercontent.com/12714889/200668905-d2c6bd70-11ec-4012-a61f-b2f5e7a40569.png">

**Reply by koksikus** [reply]
I updated Directus to the latest version and the error still persists. Note that this **only** happens when you **create a new record**. If it is created then there is no this error. 
When I cleared this values, the message that appears is always "Choices configured incorretly". The rules in the remaining scope, such as hide the field, also do not work when creating a new record in the default status, changing it the rules start working. By clicking on the default status the rule does not work.

**Comment by rijkvanzanten** [comment]
@ConnorSimply might giving this a shot to confirm/deny it's a bug?

**Reply by connorwinston** [reply]
Will take a look today!

**Comment by tonyellow** [comment]
I experienced the same problems; [https://github.com/directus/directus/discussions/16370](https://github.com/directus/directus/discussions/16370)

**Comment by koksikus** [comment]
should i do any other tests?  Or maybe I set something wrong, but it still occurs...

**Comment by koksikus** [comment]
This still happens in the latest version 9.21.0. I think it's a bug.

**Comment by makote** [comment]
I have the same behavior on 9.18 

A condition on a related field stops working when the field with the condition is placed inside of a group.
